### PR TITLE
Fix Missing Navigation in Reporting

### PR DESCRIPTION
**Pull request by nvasileiadis**
Issue: #672

**Before:**
lexicon-reports

**After:**
lexicon-reports-fix

### CODE REVIEW

**Comment by nvasileiadis** [comment]
# Breaking change quick checklist

**This is not an exhaustive list of what constitutes a breaking change**

| Does this pull request... | Yes / No |
| ---- | ---- |
| require changes to `main.js` | No |
| require changes to `index.js` | No |
| require changes to `pulsar.scss` | No  |
| require changes to `gruntfile.js` | No |
| require changes to `package.json` (not including dev dependencies) | No |
| require changes to `base.html.twig` (or other core views like header/footer) | No |
| require new arguments to be passed to a js component | No |
| require markup changes to maintain functionality | No |

**Comment by Stanton** [comment]
Fixed a handful of issues/regressions which were affecting the reporting UI

# Before
screen shot 2017-10-12 at 12 15 48

# After 
screen shot 2017-10-12 at 12 15 32

File: stylesheets/_layout.tabs.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .content-main {
     position: relative;
     display: table;
+    height: 100%;
```
Comment (Stanton) [inline stylesheets/_layout.tabs.scss]: @jadu/pulsar ⚠️ Possibility of unintended consequences, had no effect in lexicon bit shout up if this concerns you

File: stylesheets/_layout.tabs.scss [hunk 2]
```diff
@@ -88,6 +89,7 @@
 }
 
 .tabs__content {
+    height: 100%;
```
Comment (Stanton) [inline stylesheets/_layout.tabs.scss]: @jadu/pulsar ⚠️ Possibility of unintended consequences, had no effect in lexicon bit shout up if this concerns you

File: stylesheets/_layout.tabs.scss (hunk 1 above)
Comment (nvasileiadis) [reply stylesheets/_layout.tabs.scss]: Checked it in CMS and down to IE9. It seems to work properly without breaking anything.

File: stylesheets/_layout.tabs.scss (hunk 2 above)
Comment (nvasileiadis) [reply stylesheets/_layout.tabs.scss]: Checked it in CMS and down to IE9. It seems to work properly without breaking anything.

Comment (jamesjacobs) [reply stylesheets/_layout.tabs.scss]: Yeah, this looks potentially concerning. Will need to test in CXM.

File: stylesheets/_layout.tabs.scss (hunk 1 above)
Comment (jamesjacobs) [reply stylesheets/_layout.tabs.scss]: Tested in CXM all good.

File: stylesheets/_layout.tabs.scss (hunk 2 above)
Comment (jamesjacobs) [reply stylesheets/_layout.tabs.scss]: Tested in CXM all good.

File: stylesheets/_component.forms.scss [hunk 3]
```diff
@@ -311,8 +311,8 @@ fieldset {
 // * flush form groups also use the 12 column grid
 
     // default width
-    .form__control,
-    .form__control + .select2,
+    .form__group .form__control,
```
Comment (jamesjacobs) [inline stylesheets/_component.forms.scss]: This breaks inline checkboxes - will explore a fix.

Comment (jamesjacobs) [reply stylesheets/_component.forms.scss]: I've fixed this below, along with a FF width issue on checkboxes and radios

**Pull request merged**